packages/frontend/src/components/Card/Card.styled.ts

```diff
@@ -0,0 +1,115 @@
+import styled, { css } from "styled-components";
+
+import { Color } from "src/enums/color.enum";
+import { FontSize } from "src/enums/fontsize.enum";
+import { FontWeight } from "src/enums/fontweight.enum";
+import svg from "react-inlinesvg";
+import { rgba } from "polished";
+import { Link } from "react-router-dom";
+import { zIndex } from "src/enums/zIndex.enum";
+import { Breakpoint } from "src/enums/breakpoint.enum";
+import { HeartIconTypes, ImageContainerTypes } from "./Card.types";
+
+export const ImageContainer = styled.div<ImageContainerTypes>`
+  position: relative;
+  height: 140px;
+  width: 100%;
+  max-width: 100%;
+  border-radius: 10px;
+  background-image: ${({ bgImage }): string =>
+    bgImage
+      ? `url(${bgImage})`
+      : "url(http://i.wpimg.pl/1000x0/products.wpcdn.pl/photos/11124.jpg)"};
```

Comment:
nie wrzucaj fotek w taki sposób jak fotka zniknie spod tego odresu to bedzie pusto